sdk/deviceupdate/azure-iot-deviceupdate/src/samples/README.md

```diff
@@ -1,39 +1,28 @@
 # Device Update for IoT Hub Samples
 
-Samples under `generated` package are auto generated sample for you to refer how variables should be provided. 
-`ListDevicesSample` and `GetUpdateNotFoundSample` are hand-written samples that can be run directly when you configured correct environment variables.
+Samples `ListUpdatesSample`, `GetUpdateSample` and `GetDeviceSample` are hand-written samples that can be run directly 
+when you configured correct environment variables.
 
 ## Running the samples
 
-To run `ListDevicesSample` sample, you need to compile console Samples project. The executable retrieves arguments from the following
+To run the samples, you need to compile console Samples project. The executable retrieves arguments from the following
 environment settings:
 
-| Environment variable   | Description          |
-| ---------------------- | -------------------- |
-| AZURE_TENANT_ID        | AAD tenant id        |
-| AZURE_SUBSCRIPTION_ID  | AAD subscription id  |
-| AZURE_CLIENT_ID        | AAD client id        |
-| AZURE_CLIENT_SECRET    | AAD client secret    |
-| AZURE_ACCOUNT_ENDPOINT | ADU account endpoint |
-| AZURE_INSTANCE_ID      | ADU instance id      |
-
+| Environment variable         | Description          |
+|------------------------------|----------------------|
+| AZURE_TENANT_ID              | AAD tenant id        |
+| AZURE_SUBSCRIPTION_ID        | AAD subscription id  |
+| AZURE_CLIENT_ID              | AAD client id        |
+| AZURE_CLIENT_SECRET          | AAD client secret    |
+| AZURE_ACCOUNT_ENDPOINT       | ADU account endpoint |
+| AZURE_INSTANCE_ID            | ADU instance id      |
+|                              |                      |
+| DEVICEUPDATE_UPDATE_PROVIDER | fabrikam             |
+| DEVICEUPDATE_UPDATE_NAME     | vacuum               |
+| DEVICEUPDATE_UPDATE_VERSION  | 2022.401.504.6       |
+| DEVICEUPDATE_DEVICE          | dpokluda-test        |
 
 ## Creating Client
 
 You need to use AzureDeviceUpdateClientBuilder to create a proper valid AzureDeviceUpdateClient. This main client will
-let you access the other clients - Updates and Management. In the sample `ListDevicesSample`, it creates `DeviceManagementAsyncClient`.
-
-```java com.azure.iot.deviceupdate.DeviceManagementAsyncClient.instantiate
```

Comment:
Why delete this code snippet?

Reply:
I will admit that after I have updated the samples, the compilation failed with missing snippet. I didn't know how to fix it and removing this from the README was the only solution I came up with. It is fixed now properly. My bad, sorry.